src/logion/model/locrequest.model.ts

```diff
@@ -299,15 +314,17 @@ export class LocRequestAggregateRoot {
         };
     }
 
-    private itemViewable(item: { draft?: boolean, submitter?: { type?: string, address?: string } }, viewerAddress?: SupportedAccountId): boolean {
-        return !item.draft ||
+    private itemViewable(item: { status?: ItemStatus, submitter?: { type?: string, address?: string } }, viewerAddress?: SupportedAccountId): boolean {
+        return item.status === 'PUBLISHED' ||
+            item.status === 'ACKNOWLEDGED' ||
```

Comment:
The above is only true for the certificate.

Reply:
Yes, but the remaining of the condition did not change, visibility is granted to owner, submitter and requester.